electrumsv/gui/qt/network_dialog.py

```diff
@@ -1222,29 +1186,18 @@ def contextMenuEvent(self, event: QContextMenuEvent) -> None:
             return
         entry = cast(ServerListEntry, items[0].data(Roles.ITEM_DATA))
 
-        def use_as_server(auto_connect: bool) -> None:
-            nonlocal entry
-            assert entry.data_electrumx is not None
-            try:
-                self._parent_tab._parent.follow_server(entry.data_electrumx, auto_connect)
-            except Exception as e:
-                MessageBox.show_error(str(e))
+        # TODO(1.4.0) - Integrate non-electrumx servers with this
```

Comment:
This TODO comment has no classifier. All must have a one or two word classifier used for all related comments so that all the TODO comments can be easily associated with a topic.